ArduCopter/telemetry.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Copter.h"
+
+//IBus Telemetry (i.e. for fs-i6ab)
+void Copter::init_ibus_telem(){
```

Comment:
You should be able to add these sensors in the ibus_telem lib rather than in copter

Reply:
I wanted to do that, but in that way I didn't had access to the flight mode. I only have that in copter.
Any suggestion on that?

Reply:
AP_Vehicle knows the mode and has a singleton, https://github.com/ArduPilot/ardupilot/blob/164da261f10f0f6559330cf1d2dba875dd757eea/libraries/AP_Vehicle/AP_Vehicle.h#L81